### ISSUE TITLE
genome_base should be picked up from genome

### ISSUE THREAD

**Issue by szilvajuhos** [SciLifeLab/Sarek#538]
When running the flow like:
`nextflow run ${SAREKDIR}/default/main.nf --step preprocessing -profile singularityPath --project sens2017102 --sample data.tsv  --containerPath ${SAREKDIR}/containers` 

it fails, missing genome_base. 

Workaround: defining genome_base like:

`nextflow run ${SAREKDIR}/default/main.nf --step preprocessing -profile singularityPath --project sens2017102 --sample data.tsv  --containerPath ${SAREKDIR}/containers --genome_base /sw/data/uppnex/ToolBox/hg38bundle`

**Comment by MaxUlysse** [comment]
OK, thanks for reporting that...
I definitively need to check and debug everything and iron out the pipeline making the final 1.3 release.

**Comment by MaxUlysse** [comment]
Not sure if fixed or not, so I'm reopeing

**Comment by MaxUlysse** [comment]
Definitively fixed

**Comment by alneberg** [comment]
I'm still seeing this issue for awsbatch with iGenomes using 
```--genome iGRCh38``` and the config states:

```groovy
genome_base = params.genome == 'iGRCh37' ? "s3://ngi-igenomes/igenomes/Homo_sapiens/GATK/GRCh37" : params.genome == 'iGRCh38' ? "s3://ngi-igenomes/igenomes/Homo_sapiens/GATK/GRCh38" : "s3://sarek-references/small"
```

Any idea what is causing this?

**Comment by apeltzer** [comment]
The same happens if you e.g. use any other profile that already sets the variable (e.g. `binac`): It won't pick up the files required unless you specify it on the CLI again manually.

**Comment by MaxUlysse** [comment]
fixed in #711
